src/plugins/unified_search/public/query_string_input/filter_editor_wrapper.tsx

```diff
@@ -15,7 +15,7 @@ import { IDataPluginServices } from '@kbn/data-plugin/public';
 import type { DataView } from '@kbn/data-views-plugin/public';
 import { FILTER_EDITOR_WIDTH } from '../filter_bar/filter_item/filter_item';
 import { FilterEditor } from '../filter_bar/filter_editor';
-import { fetchIndexPatterns } from './fetch_index_patterns';
+import { fetchDataViews as fetchIndexPatterns } from './fetch_data_views';
```

Comment:
Why are we renaming it to fetchIndexPatterns and don't use fetchDataviews instead?